ecosystem.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  apps: [
```

Comment:
Also we need 3 apps configurations, for staging, prod and PR

Reply:
yes yes, I am working on this finding some suitable settings for the corresponding.

Reply:
Why do you need three configs for PM2 ecosystem config? You can have same app config used in all three environments na?

Reply:
We have different env and app name for all 3. And that env's are subject to change during development time so was planning to have everything separate for now. Can keep one after the first production release.